### PR TITLE
Key terms glossary

### PR DESCRIPTION
**Pull request by alkaitagi**

### CODE REVIEW

**Review by illright — APPROVED** [review]

File: documents/glossary.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+| **User presence**     | State of active engagement. Corresponds to the engagement lelvels 0-1.                                              |
+| **User absence**      | State of lack of engagement. Corresponds to the engagement levels 2-4.                                              |
+| **Tracking sequence** | Present/absent timeline used for representing user's activities over a period of time.                              |
+| **Storage format**    | Data fields and their types used for storing recoding sequence. There are 2 such formats - for SQL and File stores. |
+
+## Technical terms
+
+### Base classes
+
+|                    |                                                                          |
```
Comment (aabounegm) [inline documents/glossary.md]: Where are the headings?

File: documents/glossary.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+## Conceptual terms
+
+|                       |                                                                                                                     |
+| --------------------- | ------------------------------------------------------------------------------------------------------------------- |
+| **Gaze code**         | Representation of user's gaze, encoded as 3 states (workspace, elsewhere, absent)                                   |
+| **Voice code**        | Representation of user's voice, encoded as 2 states (present, absent).                                              |
+| **Interaction state** | Estimation of user's activity at any given time based on their gaze and speech inputs.                              |
+| **Engagement level**  | Scale of user's involment into their working process. Consists of 5 levels mapping to the interaction states.       |
+| **User presence**     | State of active engagement. Corresponds to the engagement lelvels 0-1.                                              |
```
Comment (aabounegm) [inline documents/glossary.md]:
```suggestion
| **User presence**     | State of active engagement. Corresponds to the engagement levels 0-1.                                              |
```

File: documents/glossary.md (hunk 1 above)
Comment (alkaitagi) [reply documents/glossary.md]: Term / Description. I think it is clear and no explicit label is needed.

Comment (aabounegm) [reply documents/glossary.md]: It is clear, but it renders an empty heading row in the markdown which just doesn't look right)

**Review by aabounegm — APPROVED** [review]

**Review by kuredoro — APPROVED** [review]

**Review by IamMaxim — APPROVED** [review]

**Pull request merged**